exercises/series/series_test.exs

```diff
@@ -0,0 +1,52 @@
+if !System.get_env("EXERCISM_TEST_EXAMPLES") do
+  Code.load_file("series.exs", __DIR__)
+end
+
+ExUnit.start
+ExUnit.configure exclude: :pending, trace: true
+
+defmodule StringSeriesTest do
+  use ExUnit.Case
+
+  test "slices of size 1" do
```

Comment:
we usually have  a commented out pending tag here to demonstrate how to enable the next tests.` # @tag :pending`